index.html

```diff
@@ -32,14 +32,79 @@
 </section>
 
 <section>
-  <div class="container-fluid raml-editor d-none d-sm-block">
-    <div class="row">
-      <a class="pull-right d-sm-none d-md-block playground-btn try-it-out-btn" href="">Try It Out<i class="fas fa-expand"></i></a>
-      <div class="col-md-12 col-sm-12 editor-block padd_00">
-        <div id="container" style="width:100%;height:519px;">
+  <div class="container-fluid d-none d-sm-block" id="demo-root-container">
+
+    <div class="container-fluid d-none d-sm-block" id="demo-message">
+      <h2>A simple but powerful syntax for modelling APIs</h2>
+      <h3>RAML enables rapid development of APIs using an approachable syntax<br>which can scale from hobby project to enterprise application</h3>
+    </div>
+
+    <div class="container-fluid raml-editor d-none d-sm-block">
+      <div class="row">
+        <div class="col-md-12 col-sm-12 editor-block padd_00">
+
+          <!-- Editor code tips -->
+          <div id="editor-tips-wrapper">
+
+            <div class="tip-wrapper" data-startline="2" data-lastslines="3">
+              <div class="tip-background"></div>
+              <div class="tip-text">
+                <p>Name your API, specify its version and base URL</p>
+              </div>
+            </div>
+
+            <div class="tip-wrapper" data-startline="9" data-lastslines="3">
+              <div class="tip-background"></div>
+              <div class="tip-text">
+                <p>Specify reusable types to avoid</p>
```

Comment:
to avoid what?

Reply:
duplication and redundancy :)

Thank for the catch. I've fixed it.